keyboards/gergo/rules.mk

```diff
@@ -0,0 +1,58 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+
+# Source includes
+SRC += matrix.c i2c_master.c
+#OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Hardware info
+MCU = atmega32u4
+F_CPU = 16000000
+ARCH = AVR8
+BOOTLOADER = atmel-dfu
+F_USB = $(F_CPU)
+LAYOUTS = gergo
```

Comment:
Line 16 should be deleted. This rule is only for the [Community Layouts](https://docs.qmk.fm/#/feature_layouts) feature.